docs/skill.md

```diff
@@ -1,11 +1,13 @@
-An agent developer writes skill code that the framework can call.
+<!-- adding a running example, especially handler, behaviour, task part is useful. I do not mean example of code, but example of the type of things you would do, for instance, in task vs behaviour -->
```

Comment:
Yes, that will be added. Please open an issue if you want but remove comment here.